configure.ac

```diff
@@ -24,6 +24,8 @@ m4_ifndef([AS_VAR_COPY],
     [m4_define([AS_VAR_COPY],
        [AS_LITERAL_IF([$1[]$2], [$1=$$2], [eval $1=\$$2])])])
 
+OPAL_TOP_BUILDDIR="`pwd`"
```

Comment:
OPAL_SETUP_RPATH needs this to be set to the build path because it calls `libtool --config` just before the rpath parsing stuff (OMPI does the same thing in [their configure.ac file](https://github.com/open-mpi/ompi/blob/master/configure.ac#L185)).  Maybe this line should be moved right before the OPAL_SETUP_RPATH call in our configure.ac to make it seem less out of place?